app/src/main/res/values/strings.xml

```diff
@@ -517,4 +516,82 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="questList_disabled_in_country">Never shown in %s</string>
     <string name="quest_toiletAvailability_rest_area_title">Does this rest area have a toilet?</string>
     <string name="quest_maxspeed_name_title">What is the speed limit sign for %s?</string>
+
+    <string name="quest_buildingType_title">"What kind of building is this?"</string>
+    <string name="quest_building_type_generic_building_confirmation">Are you sure that you can not determine the building type more specifically?</string>
+    <string name="quest_building_type_invalid_value">Please select a more specific value! The value you selected is not valid.</string>
+    <string name="quest_buildingType_answer_input_description">Can not determine more specifically</string>
+    <string name="quest_buildingType_comment_title">Describe the building</string>
+    <string name="quest_buildingType_comment_description">Be concise and write a short description of what the intention of the building is</string>
+    <string name="quest_buildingType_comment_emptyAnswer">You need to describe what you can see. If you have no idea what the building is used for, you can make a guess.</string>
+
+    <string name="quest_buildingType_residential">Residential</string>
+    <string name="quest_buildingType_house">House</string>
+    <string name="quest_buildingType_apartments">Apartments</string>
+    <string name="quest_buildingType_apartments_description">building arranged into individual dwellings, may have retail outlets on the ground floor, e.g flats, tenements</string>
+    <string name="quest_buildingType_detached">Detached house</string>
+    <string name="quest_buildingType_detached_description">free-standing house</string>
+    <string name="quest_buildingType_terrace">Terraced house</string>
+    <string name="quest_buildingType_terrace_description">row of residential dwellings, each with its own entrance, also known as row-house or townhouse</string>
+    <string name="quest_buildingType_hotel">Hotel</string>
+    <string name="quest_buildingType_dormitory">Dormitory</string>
+    <string name="quest_buildingType_houseboat">Houseboat</string>
+    <string name="quest_buildingType_bungalow">Bungalow</string>
+    <string name="quest_buildingType_bungalow_description">single-storey flat house, holiday cottage</string>
+    <string name="quest_buildingType_static_caravan">Static caravan</string>
+
+    <string name="quest_buildingType_commercial">Commercial</string>
+    <string name="quest_buildingType_commercial_description">building where general commercial activities take place</string>
+    <string name="quest_buildingType_industrial">Industrial</string>
+    <string name="quest_buildingType_industrial_description">building where industrial process takes place, e.g. factory</string>
```

Comment:
is it also fitting service buildings on railway station (storage & repair of locomotives, housing of various railway machinery and equipment etc).